integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java

```diff
@@ -916,17 +956,20 @@ public void testDynamicServerLifeCycleWithoutAdmin() {
       logger.info("Server[" + serverName + "] pod is initialized");
 
       // (re)Start Start the admin
-      patchServerStartPolicy(
-             "/spec/adminServer/serverStartPolicy", "IF_NEEDED");
+      assertTrue(patchServerStartPolicy(domainUid, domainNamespace,
+            "/spec/adminServer/serverStartPolicy", "IF_NEEDED"),
+             "Failed to patch adminServer's serverStartPolicy to IF_NEEDED");
       checkPodReadyAndServiceExists(
              adminServerPodName, domainUid, domainNamespace);
-      logger.info("AdminServer restart success");
+      logger.info("administration server restart success");
 
       checkPodReadyAndServiceExists(serverPodName, domainUid, domainNamespace);
       logger.info("(re)Started [" + serverName + "] on admin server restart");
     } finally {
       // restart admin server
-      patchServerStartPolicy("/spec/adminServer/serverStartPolicy", "IF_NEEDED");
+      assertTrue(patchServerStartPolicy(domainUid, domainNamespace,
```

Comment:
you have the same code both inside try and in finally blocks, keep it at one place(finally).

Reply:
The code inside the try/catch block is a part of test, which verifies that the managed server get started once the admin server is (re) started. The finally block is for if the test fails for any other reason before reaching the above scenario.

Reply:
finally block gets executed in all cases, you can just keep the code in finally. No need to keep the code in both try and finally.

Reply:
I know. I want to start the stop/start the admin server in my test. The admin server start in finally block is void if the test pass.